Batch.py

```diff
@@ -58,28 +61,28 @@ def data_into_batch(self, data):
         camera_data = camera_data.cuda().float() / 255. - 0.5
         camera_data = torch.transpose(camera_data, 0, 2)
         camera_data = torch.transpose(camera_data, 1, 2)
-        self.camera_data = torch.cat((self.camera_data,
-                                      torch.unsqueeze(camera_data, 0)), 0)
+        self.camera_data[data_number, :, :, :] = camera_data
 
         # Convert Behavioral Modes/Metadata to PyTorch Ready Tensors
-        metadata = torch.FloatTensor().cuda()
-        zero_matrix = torch.FloatTensor(1, 1, 23, 41).zero_().cuda()
-        one_matrix = torch.FloatTensor(1, 1, 23, 41).fill_(1).cuda()
+        metadata = torch.FloatTensor(128, 23, 41).cuda()
+        zero_matrix = torch.FloatTensor(23, 41).zero_().cuda()
+        one_matrix = torch.FloatTensor(23, 41).fill_(1).cuda()
+        metadata_count = 127
         for cur_label in ['racing', 'caffe', 'follow', 'direct', 'play',
                           'furtive']:
             if cur_label == 'caffe':
                 if data['states'][0]:
-                    metadata = torch.cat((one_matrix, metadata), 1)
+                    metadata[metadata_count, :, :] = one_matrix
```

Comment:
The matricies were being prepended to the final output in the previous code, is this still happening here? I can see it when you pad the empty tensor but I want to make sure this is consistent.